austation/code/modules/research/techweb/all_nodes.dm

```diff
@@ -16,6 +16,15 @@
 
 /////////////////////////Mining tech/////////////////////////
 
+/datum/techweb_node/basic_mining
+	id = "basic_mining"
+	display_name = "Mining Technology"
+	description = "Better than Efficiency V."
+	prereq_ids = list("engineering", "basic_plasma")
+	design_ids = list("drill", "superresonator", "triggermod", "damagemod", "cooldownmod", "rangemod", "ore_redemption", "mining_equipment_vendor", "cargoexpress", "plasmacutter", "minebot_fab") //e a r l y    g a  m e)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	export_price = 5000
+
```

Comment:
you're joking, right? use the modular options I made to add new items to existing nodes rather than overriding THE ENTIRE DATUM
***ARGHH***
```suggestion
/datum/techweb_node/basic_mining
	austation_design_ids = list("minebot_fab")

```

Reply:
I blame elon.
I asked him how to use the new techweb stuff and he said he didn't know.

Reply:
not my fault

Reply:
I'm joking